test/http_tiles.cc

```diff
@@ -0,0 +1,224 @@
+#include "test.h"
```

Comment:
this is an ingenous little test if i do say so myself :)  it runs an http file server using prime_server to serve up the utrecht tiles we use for lots of other unit tests.